src/commands/junit/api.ts

```diff
@@ -56,7 +57,7 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
   form.append('event', JSON.stringify(custom), {filename: 'event.json'})
 
   form.append('junit_xml_report_file', fs.createReadStream(jUnitXML.xmlPath).pipe(createGzip()), {
-    filename: `${replaceForwardSlashes(fileName)}.xml.gz`,
+    filename: `${uuidv4()}.xml.gz`,
```

Comment:
We want to apply this also to the filename in sarif, don't we?

Reply:
Is it the same as junit that EvP handles the uniqueness of the upload? I am not very knowledgeable about that part

Reply:
yes the upload to s3 is common